Database/Corrections/sodQuestFixes.lua

```diff
@@ -2358,6 +2358,33 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -424992,
             [questKeys.zoneOrSort] = sortKeys.ROGUE,
         },
+        [90139] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {{1161,1162,1163,1164}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
```

Comment:
The Skullthumper Troggs are level 13-14, so I'd say this should also be a level 14 quest.
```suggestion
            [questKeys.questLevel] = 14,
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5435/commits/f2044c1371ec0bd14f39dc543cd494b29bae3ac5

I've done it at lower level than that, and the majority of people that go to that quest area are level 10-12. The Skullthumpers can be dealt with at level 11-12, so assumed that at least letting users know it was in the area would be helpful before level 14?

Reply:
The rune will still show it is just the "recommended level", same as Blizzard does it for _most_ quests.

Reply:
Ah yes. That's the whole point of 'requiredLevel'.